presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -185,6 +225,7 @@ public TaskExecutor(
             int minDrivers,
             int guaranteedNumberOfDriversPerTask,
             int maximumNumberOfDriversPerTask,
+            TaskPriorityTracking taskPriorityTracking,
```

Comment:
It feels like priority tracking should be an implementation detail of the TaskExecutor. I don't mind exposing it if it is ever actually needed though. Since now it is not needed - i would prefer keeping it as is.